.github/workflows/beta-release.yaml

```diff
@@ -0,0 +1,76 @@
+name: Beta Release
+
+on:
+  workflow_call:
+
+jobs:
+  beta_release:
+    runs-on: ubuntu-latest
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'yarn'
+
+      - name: Install dev dependencies
+        run: yarn install
+
+      - name: Setup npm credentials file
+        run: echo "//registry.npmjs.org/:_authToken=$NODE_AUTH_TOKEN" >> .npmrc
+
+      - name: Setup git credentials
+        run: |
+          git config --global user.name 'Auto Release Bot'
+          git config --global user.email 'auto-release-bot@users.noreply.github.com'
```

Comment:
I believe "Auto Release Bot" is not recognized as an actual user, so it blocks the checks. Please try omitting this step, or setting to the committer to "github-actions[bot]".

Reply:
this step is needed to save the current beta version into the package (to commit the changes), I can also set it to my user if the suggestion doesn't work, let's try